docs/contributing/documentation_guidelines.md

```diff
@@ -13,6 +13,66 @@ Wagtail’s documentation uses a mixture of [Markdown](https://myst-parser.readt
 
 Here are formats we encourage using when writing documentation for Wagtail.
 
+## Writing style guide
+
+To ensure consistency in tone and language, follow the [Google developer documentation style guide](https://developers.google.com/style) when writing the Wagtail documentation.
+
+## Writing style linter
+
+Before submitting a pull request, use [Vale](https://vale.sh/) to check if your documentation adheres to the Google developer documentation style guide. Follow these steps to check your documentation with Vale:
+
+1.  Copy and paste your Wagtail documentation content into a new markdown file located in a separate directory.
+2.  Open your command line terminal and navigate to the separate directory that has your new markdown file.
+3.  Install Vale using a package manager suitable for your operating system, such as [Homebrew](https://brew.sh/) for macOS or [Chocolatey](https://phoenixnap.com/kb/chocolatey-windows) for Windows.
+    -  For Windows:
+    
+    ```doscon
+    choco install vale
+    ```
+    
+    -  For macOS:
+    
+    ```sh
+    brew install vale
+    ```
+
+4.  Verify the successful installation of Vale by running the command:
+
+```sh
+vale -v
+```
+
+This should display the version of Vale you installed.
+5.  Create a configuration file named `.vale.ini` in the same directory.
+6.  Paste the following contents into your `.vale.ini` configuration file:
+
+```txt
+StylesPath = styles
+MinAlertLevel = suggestion
+Packages = Google
+
+[*]
+BasedOnStyles = Vale, Google
+```
```

Comment:
A bit of feedback on using this, I gave this a go with my PR https://github.com/wagtail/wagtail/pull/10590

- It seems to check inline code snippets e.g. `<a href=""...></a>` was checked for things like brackets which is incorrect, same with code blocks.
- It flags the myst internal refs as using too many brackets e.g. `(some_ref)=`, which it should ignore.
- It says `admin` should be `administrator` which would flag a lot of false errors in our existing documentation. Similarly it flags `application` should be `app` which may not be suitable.
- This base config misses a lot of spelling such as `Stimulus` and `Draftail` as proper nouns.
- Re-writing a lot of things into active (not passive) voice is hard, probably just my bad writing skills, I envisage many others will struggle with this.
- It did not seem to flag any issues with UK spelling (e.g. using `behaviour` not `behavior`), not sure if that's something to consider if we want to also adopt US spelling.

I think the vale checks are great and will keep trying it out but it may need some more work for us to have a baseline reliable config that's a good starting point for the majority of our documentation.

Reply:
One more note, not sure if it's my setup but after using Vale with VScode my mac slowed down a lot.

It appears to take up a large amount of memory.

<img width="679" alt="Screenshot 2023-07-17 at 8 29 26 am" src="https://github.com/wagtail/wagtail/assets/1396140/245f370c-a805-497c-a39e-261abbadc2b0">